torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -39,6 +39,7 @@ static std::unordered_map<std::string, ParameterType> type_map = {
   {"Device", ParameterType::DEVICE},
   {"Stream", ParameterType::STREAM},
   {"std::string", ParameterType::STRING},
```

Comment:
Note to self: can I remove this?

Reply:
Answer: not yet, because `Device` takes a `std::string`. https://github.com/pytorch/pytorch/blob/a0ac80ec768f29c46a2d4397d491d86727b7760c/torch/csrc/Device.cpp#L54

Reply:
Actually, we need to keep `std::string` for the output of `Event.ipc_handle`